web/src/beta/features/Editor/Visualizer/CanvasArea/hooks.ts

```diff
@@ -227,7 +231,7 @@ export default ({ sceneId, isBuilt }: { sceneId?: string; isBuilt?: boolean }) =
     onBlockRemove,
     onBlockInsert,
     onWidgetUpdate,
-    selectWidgetArea,
+    selectWidgetArea: selectedWidgetAreaVar,
```

Comment:
I think you are assigning the getter to the setter here. You should assign seelctedWidgetAreaVar to selectedWidgetArea above on line 221:

 This might be why I am getting inconsistent results when I try changing padding on multiple sides.